actionpack/test/controller/request_forgery_protection_test.rb

```diff
@@ -710,6 +714,46 @@ def test_rejects_token_for_incorrect_method
     end
   end
 
+  def test_rejects_token_for_incorrect_method_button_to
+    get :button_to, params: { form_method: 'delete' }
+
+    form_token = nil
+    assert_select 'input[name=custom_authenticity_token]' do |elts|
```

Comment:
Nokogiri doesn't like selecting on params without quotes, switch to: `'input[name="custom_authenticity_token"]'`